app/controllers/publishers_controller.rb

```diff
@@ -451,6 +451,9 @@ def create_site_banners
         channel.site_banner = SiteBanner.new_helper(current_publisher.id, channel.id)
       end
     end
+    if current_publisher.channels.length.zero?
+      current_publisher.update(default_site_banner_mode: true)
```

Comment:
let's do this 
```ruby
current_publisher.update(default_site_banner_mode: true) if current_publisher.channels.length.zero?
```

Reply:
how bow dis? 

```
    if current_publisher.channels.length.zero?
      current_publisher.update(default_site_banner_mode: true)
    else
      current_publisher.channels.each do |channel|
        channel.site_banner = SiteBanner.new_helper(current_publisher.id, channel.id) if channel.site_banner.nil?
      end
    end
```

Reply:
sorry! I didn't see that it had an else statement. 👍